contracts/stellar-axelar-operators/src/contract.rs

```diff
@@ -87,7 +88,7 @@ impl AxelarOperatorsInterface for AxelarOperators {
     }
 }
 
-impl AxelarOperators {
-    // Modify this function to add migration logic
-    const fn run_migration(_env: &Env, _migration_data: ()) {}
+impl CustomMigratableInterface for AxelarOperators {
+    type MigrationData = ();
+    fn __migrate(_env: &Env, _migration_data: ()) {}
```

Comment:
it's different than others - should it be the same?
```suggestion
    fn __migrate(_env: &Env, _migration_data: Self::MigrationData) {}
```

Reply:
yes thanks, I knew I did that somewhere but just couldn't find it